src/test/java/net/snowflake/client/jdbc/ResultSetJsonVsArrowMultiTZIT.java

```diff
@@ -65,13 +64,12 @@ public ResultSetJsonVsArrowMultiTZIT(String queryResultFormat, String timeZone)
   }
 
   private Connection init(String table, String column, String values) throws SQLException {
-    Connection con = getConnection(BaseJDBCTest.DONT_INJECT_SOCKET_TIMEOUT);
-    try (Statement statement = con.createStatement()) {
+    try (Statement statement = connection.createStatement()) {
       statement.execute("alter session set jdbc_query_result_format = '" + queryResultFormat + "'");
       statement.execute("create or replace table " + table + " " + column);
       statement.execute("insert into " + table + " values " + values);
     }
-    return con;
+    return connection;
```

Comment:
do we need to have connection returned here? can we change the return type to void and in the tests still use the same connection?